src/utils/get-config.js

```diff
@@ -2,12 +2,16 @@ import { readFileSync, statSync } from 'fs';
 import path from 'path';
 import resolveFileUrl from './resolve-file-url';
 import isAbsolute from './is-absolute';
+import isAsarPath from './is-asar-path';
 import CONSTANTS from '../constants';
 
 
 const PROTOCOL_RE = /^([\w-]+?)(?=\:\/\/)/;
 
 export default function (id, mainPath) {
+    if (isAsarPath(mainPath))
```

Comment:
`mainPath` is a path to a config file. We won't allow storing config files in ASAR archives, so I guess we can remove it?